.craft.yml

```diff
@@ -41,3 +41,5 @@ targets:
       maven:io.sentry:sentry-apollo:
       maven:io.sentry:sentry-jdbc:
       maven:io.sentry:sentry-graphql:
+      maven:io.sentry:sentry-android-navigation:
+      maven:io.sentry:sentry-compose:
```

Comment:
You have to add them first (manually) to https://github.com/getsentry/sentry-release-registry
otherwise, releases will fail.

Reply:
shouldn't we first push this to maven central and then merge the release registry? or it's other way around?

Reply:
It is the other way around, but if you merge this, you won't be able to release (push to maven central), because the release registry does not exist, craft will fail for non-preview releases.

Reply:
Alright, thanks for explaining 👍 will make sure the release registry is out first